src/config/default.properties

```diff
@@ -10,14 +10,15 @@ orderly_test_github_org=vimc-auth-test
 docker_host=tcp://localhost:2375
 
 app_url=http://localhost:8081
-app_port=8081
+app_port=8888
```

Comment:
I changed this because when you run the proxy locally it spins up the reporting API on port 8081 :woman_facepalming: In general we probably want a different port for local dev for this reason, the real port it gets deployed on will be configured by the deploy tool anyway.

Reply:
Ha, of course!